### ISSUE TITLE
Templates/Samples Update

### ISSUE THREAD

**Issue by mtg1986** [IBVSoftware/CollaBoard-Feedback#581]
For all the templates and samples there isn't an update date set. It says:

"Code XY"
"Created on YZ updated on n/a"

Therefor we'd need an update date to be set.

![template date](https://user-images.githubusercontent.com/19570958/40102727-9014b2b6-58eb-11e8-977f-57a4fde32da0.JPG)

Version 2.0.22.0

Tested on Surface Pro and Surface Hub

**Comment by corradocavalli** [comment]
CollaborationService.GetTemplateProjectsAync result doesn't update LastUpdated field, that's why is reported as N/A.

**Comment by corradocavalli** [comment]
Fixed on DEV environment, client app is not involved in this case.

**Comment by gsantopaolo** [comment]
@gsantopaolo mention to trigger vsts import

**Comment by gsantopaolo** [comment]
@gsantopaolo mention to trigger vsts import

**Comment by gsantopaolo** [comment]
@corradocavalli  con you confirm is fixed only on dev. so this means we have to update server as well release on production?

**Comment by corradocavalli** [comment]
Yes,change has been done and tested with @kasuken with DEV server.

**Comment by gsantopaolo** [comment]
Ok so this means that when we deploy we have to deploy client and server

**Comment by mtg1986** [comment]
Tested with version 2.0.25 on the Surface Hub, still looks the same.

**Comment by mtg1986** [comment]
@gsantopaolo : Got a new version from Corrado 2.0.28. and tested this issue again. It still looks the same. 
![image](https://user-images.githubusercontent.com/19570958/41409584-febe4428-6fd6-11e8-82ef-c768fda9bacd.png)

**Comment by gsantopaolo** [comment]
@mtg1986  can't understand. @corradocavalli do you know anything?

**Comment by corradocavalli** [comment]
This bug has already been fixed by Ema, but we need to update the PROD server with latest bits.

**Comment by gsantopaolo** [comment]
@mtg1986  answered on a different thread. We can plan a CB production serveices updates when i'll be back from holidays..  feel free to keep this open until you get it working on prod

**Comment by agorog** [comment]
Tested on Azure On Prem with new installed services and 51 Client. t.project.lastupdate and t_project.updatedby are not updated

**Comment by mtg1986** [comment]
Tested it with 2.0.55 and the bug is still here.
